### ISSUE TITLE
extDB2 for linux gives error in log (and fails to connect)

### ISSUE THREAD

**Issue by alexnewmn** [ArmaLife/extDB2#10]
So when i want to run my altislife mission using extDB2 it is giving me error
https://hastebin.com/olidereqaf.pas

here is rpt
http://zenoxgaming.com/download/putty.log

I am running linux, ubuntu

Also i tested on a clean altislife 5.0 mission with extDB3 same information in config etc and it connects, so it must be problem with extDB2 linux, i really want to use extDB2 as my mission that have the good files etc and all stuff done is using that.

Thank you i hope this gets fixed or you can tell me what i done wrong

**Comment by Jawshy** [comment]
This is the same error as described in https://github.com/AsYetUntitled/Framework/issues/285 and https://github.com/AsYetUntitled/extDB2/issues/9. Please do not create duplicate issues, @Alexlyn1.

**Comment by alexnewmn** [comment]
Yes, but nothing was ever fixed about it, and as you said this was a issue for extDB2, not the mission. 
How is it an user error? And i asked in discord, but no one knew / couldnt help. 
I have tried the same information on extDB3 using altislife 5.0 however 5.0 isnt really what i need to run atm, also you say extDB3 is the new, but altislife 5.0 isnt stable/supposed to be used yet, perhabs we should know how to use extDB3 on the 4.0 version when its not using extDB3 by default then? @Jawshy
